docs/source-app/core_api/lightning_work/compute_content.rst

```diff
@@ -22,6 +22,15 @@ a :class:`~lightning_app.utilities.packaging.cloud_compute.CloudCompute` to your
     # Run on a fast multi-GPU machine (see specs below)
     MyCustomWork(cloud_compute=L.CloudCompute("gpu-fast-multi"))
 
+.. warning::
+     Custom base images are not supported with the default-cpu. For example:
```

Comment:
Maybe
```
Custom base images are not supported when CloudCompute is set to "default-cpu"
```
?

Reply:
+1

Reply:
Also- should this be a warning? or a note?

Reply:
Definitely a warning on the framework side

Reply:
@Felonious-Spellfire Why are we adding this warning in the CloudCompute section? The reader has not been introduced to the concept of BuildConfig at this time and it is irrelevant when working with compute.

Reply:
@awaelchli 
Because someone might come here to see what we support for compute. It saves them toggling back and forth between pages. But I can remove it if you prefer.

Reply:
Fine with me to keep, just want to make sure we don't distract readers. Btw, the warning can very well also be done at runtime.

And btw, the default setting is not "default-cpu". It's just "default".